### ISSUE TITLE
VS: When WinUI target is not physically listed first, XAML intellisense is not working properly

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno.templates#758]
### Current behavior

When a blank Uno app is created, IntelliSense for built-in resources, etc. is not working. that is even if one selects `net8.0-windowsxyz' as target in the top left dropdown. However, when the windows target is put first in order explicitly in the `.csproj`, it starts working correctly:

![image](https://github.com/unoplatform/uno.templates/assets/1075116/78bd24fa-8006-4357-9a10-e1a07a864fe6)

### Expected behavior

Should work out of the box

### Template Host

Visual Studio

### Host OS

Windows

### Template Options

Latest stable

### Template Version

```shell
Latest stable
```

**Comment by MartinZikmund** [comment]
@jeromelaban I would expect the target reordering would be enough, but it seems it is not

**Comment by dansiegel** [comment]
@MartinZikmund this is with an Uno.Sdk SingleProject? This was working so I'm a bit surprised to hear that there is an issue again.

**Comment by MartinZikmund** [comment]
Yes indeed, single project. It only works for me if I move -windowsxyz target to the first position in .csproj
